src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -151,6 +173,15 @@ public String getRepoRoot() {
         return path;
     }
 
+    public String getRepoFolderName() {
```

Comment:
Sorry, can explain why this is needed?

Reply:
Sorry should have probably explained somewhere.

Currently, we use `repoName` of `RepoLocation` as the name of the folder that repos are cloned into. However, 2 different repos can have the same `repoName`, for example all CS2103 projects will have "main" as their  `repoName`. So when we clone and analyze two repos, the second one will override the first one.

`getRepoFolderName()` uses `organization` of `RepoLocation` as well to ensure that the folder is unique across different repos.

Reply:
I see.

Anyway, to prevent the same issue as #378, let's just create a new `final` variable `repoFolderName`, which is set to this value within the constructor.

Reply:
Done.